iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -375,7 +376,7 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
             try
             {
                 await EnableTwinPatchAsync(cancellationToken).ConfigureAwait(false);
-                Twin twin = await RoundTripTwinMessage(AmqpTwinMessageType.Patch, reportedProperties, cancellationToken).ConfigureAwait(false);
+                await RoundTripTwinMessage(AmqpTwinMessageType.Patch, reportedProperties, cancellationToken).ConfigureAwait(false);
```

Comment:
Why doesn't this method have an Async suffix?

Reply:
There is a timeout based condition in this method RoundTripTwinMessage where it waits until a specific task is completed or timed out, perhaps that's why.

Reply:
I can't see where in this PR where this method lives, so it seems you didn't add it. However, it isn't named properly. Can you rename it to fix it?

Reply:
Yes I certainly did not add this method. 
I am open to any suggestions for renaming, don't have anything in mind which would fit better than what it is.

Reply:
`RoundTripTwinMessageAsync`

Reply:
I get what you are saying now. Thanks for clarifying

Reply:
Resolving comments as I go through them in my local branch.